### PR TITLE
README: Explain both hook scripts

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,21 @@ which does the equivalent of:
 
 with minor changes to the input/output format used between the commands.
 
+### openQA hook scripts - Call auto-review or investigation steps in openQA after every job is done
+
+openQA supports custom job done hook scripts that can be called whenever a job
+is done, see
+http://open.qa/docs/#_enable_custom_hook_scripts_on_job_done_based_on_result
```
Comment (Martchus) [inline README.md:72]:
```suggestion
https://open.qa/docs/#_enable_custom_hook_scripts_on_job_done_based_on_result
```

Comment (okurz) [reply README.md:72]: I see no problem with http for the static documentation webpage. Especially when one is redirected by the browser automatically in most cases anyway

Comment (Martchus) [reply README.md:72]: This was more for consistency. But I also don't care much. If you like you can merge it as-is.

Comment (okurz) [reply README.md:72]: As we have not mentioned "open.qa" anywhere else in this repo I will keep it

**Pull request merged**